test_board.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Lyngk Website</title>
+
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+  <html>
+    <body>
+      <div id="unclaimed_label">
+        <h3>
+          Unclaimed colors:
+        </h3>
+      </div>
+      <div id ='summaryandmoves'>
+        <h1>Summary of Lyngk Board 191</h1>
+
+        <p>It is p1&apos;s turn.</p>
+        <div id ='movesDiv'>
+          <table id ='moves'>
+	         <tr><th>p1</th><th>p2</th></tr>
+	         <tr><td>1 a1-b3</td><td>2 R,b4-b3</td></tr>
+	         <tr><td>3 a1-b3</td><td>4 R,b4-b3</td></tr>
+	         <tr><td>5 a1-b3</td><td>6 R,b4-b3</td></tr>
+	         <tr><td>7 a1-b3</td><td>8 R,b4-b3</td></tr>
+	         <tr><td>9 a1-b3</td><td>10 R,b4-b3</td></tr>
+	         <tr><td>11 a1-b3</td><td>12 R,b4-b3</td></tr>
+          </table>
+        </div>
+        <div id='player1claimed' class='hiddenHTML'>RK</div>
+        <div id='player2claimed' class='hiddenHTML'>BI</div>
+        <div id='boardstring' class='hiddenHTML'>|B|K|KGR||K|I|I|K|G|J|K|K|B|I|B|G|R|G|K|R|G|K|J|B|R|R|B|G|J|G|I|G|B|R|B|I|I|R|B|I|R|I|</div>
+      </div>
+      <canvas width=400 height=400 id="lyngkboard">
+        Your browser does not support the HTML5
+      </canvas>
+      <canvas width=80 height=400 id="unclaimedCanvas">
+          Your browser does not support the HTML5
+      </canvas>
+      <div id='player1div'>
+        <p>Claimed by Player 1:</p>

```

Comment:
Insert player's name (e.g., p1).  For testing, I suggest we use two player names of differing lengths.